platforms/gke/base/core/teardown.sh

```diff
@@ -32,7 +30,7 @@ cd ${ACP_PLATFORM_CORE_DIR}/initialize &&
     terraform apply -input=false tfplan || exit 1
 rm tfplan
 
-for terraservice in "${terraservices[@]}"; do
+for terraservice in $(echo "${terraservices[@]}" | tac -s " "); do
```

Comment:
Nit: you can simplify this expression by iterating on the array in reverse, without the need of using `echo` and `tac`:

```bash
for (( i = ${#terraservices[@]} - 1; i >= 0; i-- )); do
  terraservice=${terraservices[i]}
  ...
done
```

Reply:
I'm running through some tests as I work on some other use cases, and I think it might make more sense to remove the reversing of the array. This will allow for setting a custom order for destroy.